cpp/include/Ice/Proxy.h

```diff
@@ -479,16 +493,17 @@ namespace Ice
             std::function<void(bool)> sent = nullptr,
             const Ice::Context& context = Ice::noExplicitContext) const;
 
-        /// Obtains the Connection for this proxy. If the proxy does not yet have an established connection,
+        /// Gets the Connection for this proxy. If the proxy does not yet have an established connection,
```

Comment:
Do you use `Connection` for the auto-link or should it be connection?

Reply:
It should be lowercase.